### PR TITLE
Updating vMaster-rc2 images

### PR DESCRIPTION
**Pull request by keerthibingi**

### CODE REVIEW

File: charts/isdargo/rcimages-values.yaml [hunk 1]
```diff
@@ -189,7 +189,7 @@ argo-rollouts:
 auditClient:
   image:
     repository: ubi8-oes-audit-client
-    tag: v4.1.2.1-rc2
+    tag: vMaster-rc2
 ###################################################
 gate:
```
Comment (saitejaopsmx) [inline charts/isdargo/rcimages-values.yaml:194]: Need to update the gate tag as well

**Review by saitejaopsmx — COMMENTED** [review]
Please update the gate tag as well

File: charts/isdargo/rcimages-values.yaml (hunk 1 above)
Comment (keerthibingi) [reply charts/isdargo/rcimages-values.yaml:194]: Updated latest tag for gate

File: charts/isdargo/rcimages-values.yaml [hunk 2]
```diff
@@ -189,52 +189,52 @@ argo-rollouts:
...
   image:
     repository: ubi8-gate
-    tag: v4.1.2.1-rc2
+    tag: vMaster-rc2
```
Comment (saitejaopsmx) [inline charts/isdargo/rcimages-values.yaml]: Please remove the extra space

Comment (keerthibingi) [reply charts/isdargo/rcimages-values.yaml]: Done

**Pull request merged**